packages/orchestration/test/exos/local-orchestration-account-kit.test.ts

```diff
@@ -266,6 +288,38 @@ test('transfer', async t => {
     },
   });
 
+  const resultMeta = await doTransferWithMeta(aDenomAmount, dydxDest);
+  const { result: multiHopResult, meta: multiHopMeta } = resultMeta;
+
+  t.is(latestTxMsg().receiver, PFM_RECEIVER, 'defaults to "pfm" receiver');
+  t.deepEqual(JSON.parse(latestTxMsg().memo), {
+    forward: {
+      receiver: 'dydx1test',
+      port: 'transfer',
+      channel: 'channel-33',
+      retries: 3,
+      timeout: '10m',
+    },
+  });
+
+  t.deepEqual(multiHopMeta, {
+    traffic: [
```

Comment:
`multiHopMeta` looks like only 1 hop

Reply:
ok... we don't get metadata on the other hops; this test works just like the non-meta one